block-storage/src/main/scala/coop/rchain/blockstorage/FileLMDBIndexBlockStore.scala

```diff
@@ -115,7 +115,7 @@ class FileLMDBIndexBlockStore[F[_]: Monad: Sync: RaiseIOError: Log] private (
   override def get(blockHash: BlockHash): F[Option[BlockMessage]] =
     lock.withPermit(
       for {
-        indexEntryBytesOpt     <- index.get(blockHash.toDirectByteBuffer)
+        indexEntryBytesOpt     <- index.get_WARNING(blockHash.toDirectByteBuffer)
```

Comment:
why not camel case?

Reply:
It's on purpose, because this function should be removed and not used in the new code. When block store will also be stored to LMDB as we did for DAG storage this function will be removed.